Doc/library/typing.rst

```diff
@@ -365,7 +365,7 @@ need to mix dynamically and statically typed code.
 
 Contrast the behavior of :data:`Any` with the behavior of :class:`object`.
 Similar to :data:`Any`, every type is a subtype of :class:`object`. However,
-unlike :data:`Any`, the reverse is not true: :class:`object` is *not* a
+unlike :data:`Any`, the reverse is not a truthy value: :class:`object` is *not* a
```

Comment:
This is why `sed` may be a bad idea. This should stay "true".

Reply:
I have doubts about all other changes too. This PR is just a demonstration how large the change can be. And it is only a small part, there are around 1000 other potential changes, which must be made and checked manually.